src/features/offer/pages/Offer.tsx

```diff
@@ -1,51 +1,68 @@
 import { t } from '@lingui/macro'
-import { StackScreenProps } from '@react-navigation/stack'
+import { useRoute } from '@react-navigation/native'
 import React, { FunctionComponent } from 'react'
 import styled from 'styled-components/native'
 
-import { RootStackParamList } from 'features/navigation/RootNavigator'
+import { UseRouteType } from 'features/navigation/RootNavigator'
 import { LocationCaption } from 'features/offer/atoms/LocationCaption'
 import { _ } from 'libs/i18n'
 import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
 
+import { AlgoliaCategory } from '../../../libs/algolia'
```

Comment:
```suggestion
import { AlgoliaCategory } from 'libs/algolia'
```

c'est pas grand chose mais je le mets quand même